### PR TITLE
[iOS] Update Sample Server 7 to 10.8.1

### PR DESCRIPTION
**Pull request by vquach2404**
This PR updates samples that use sample server 7. The URLs have been updated and credentials have been added. For testing purposes, you will need to be on the VPN and update the hosts file on your machine. Please see `/common-samples/issues/2270` for more info and see Zack's comment about updating the hosts file. 

Note: The Naperville Electric feature service has changed slightly but will be updated to match its previous state. Display subtype feature layer, Display utility associations, and trace utility network may appear different as a result.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift [hunk 1]
```diff
@@ -25,7 +25,7 @@ class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelega
     @IBOutlet weak var statusLabel: UILabel!
     @IBOutlet weak var modeControl: UISegmentedControl!
     
-    private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
+    private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift:28]: The updated service doesn't show network or controllers. Could you verify that?

**Review by yo1995 — COMMENTED** [review]
All the update samples look fine to me, with 1 exception.

File: arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift (hunk 1 above)
Comment (yo1995) [reply arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift:28]: noshow

It looks like this to me.

**Comment by yo1995** [comment]
Please feel free to add Sarat and Phil after the merge conflict is solved.

File: arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift (hunk 1 above)
Comment (yo1995) [reply arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift:28]: Also, should we update the URLs in README?

Comment (vquach2404) [reply arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift:28]: Fixed this sample and also updated the URLs in the READMEs!

**Review by yo1995 — APPROVED** [review]
👍  Work as expected.

File: arcgis-ios-sdk-samples/Layers/Display subtype feature layer/DisplaySubtypeFeatureLayerViewController.swift [hunk 2]
```diff
@@ -54,10 +54,10 @@ class DisplaySubtypeFeatureLayerViewController: UIViewController {
     }
     
     func makeMap() -> AGSMap {
+        AGSAuthenticationManager.shared().delegate = self
```
Comment (philium) [inline arcgis-ios-sdk-samples/Layers/Display subtype feature layer/DisplaySubtypeFeatureLayerViewController.swift:57]: This seems like a very odd place for this. Could it go somewhere else, like maybe immediately before the call to `makeMap()`?

**Review by philium — APPROVED** [review]

**Review by saratchandrakarumuri — APPROVED** [review]
All the updated services work well

**Comment by yo1995** [comment]
Hi @vquach2404 would you mind put this PR on your todo list, and resolve the conflicts before next Monday?
I plan to set up the release prior to next Wednesday, so we can directly release the TestFlight build on the exact date.
Thanks.

**Review by yo1995 — APPROVED** [review]

**Pull request merged**